diff --git a/node_modules/leaflet/src/control/Control.Attribution.js b/node_modules/leaflet/src/control/Control.Attribution.js
index 1a96542..fa3700b 100644
--- a/node_modules/leaflet/src/control/Control.Attribution.js
+++ b/node_modules/leaflet/src/control/Control.Attribution.js
@@ -6,7 +6,7 @@ import * as DomEvent from '../dom/DomEvent';
 import * as DomUtil from '../dom/DomUtil';
 import Browser from '../core/Browser';
 
-var ukrainianFlag = '<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="12" height="8" viewBox="0 0 12 8" class="leaflet-attribution-flag"><path fill="#4C7BE1" d="M0 0h12v4H0z"/><path fill="#FFD500" d="M0 4h12v3H0z"/><path fill="#E0BC00" d="M0 7h12v1H0z"/></svg>';
+var ukrainianFlag = '<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="500" height="8" viewBox="0 0 12 8" class="leaflet-attribution-flag"><path fill="#4C7BE1" d="M0 0h12v4H0z"/><path fill="#AAD500" d="M0 4h12v3H0z"/><path fill="#E0BC00" d="M0 7h12v1H0z"/></svg>';
 
 
 /*
@@ -25,7 +25,7 @@ export var Attribution = Control.extend({
 
 		// @option prefix: String|false = 'Leaflet'
 		// The HTML text shown before the attributions. Pass `false` to disable.
-		prefix: '<a href="https://leafletjs.com" title="A JavaScript library for interactive maps">' + (Browser.inlineSvg ? ukrainianFlag + ' ' : '') + 'Leaflet</a>'
+		prefix: false,
 	},
 
 	initialize: function (options) {
diff --git a/node_modules/leaflet/src/control/index.js b/node_modules/leaflet/src/control/index.js
index 92d5205..600ff6a 100644
--- a/node_modules/leaflet/src/control/index.js
+++ b/node_modules/leaflet/src/control/index.js
@@ -12,6 +12,5 @@ Control.Attribution = Attribution;
 control.layers = layers;
 control.zoom = zoom;
 control.scale = scale;
-control.attribution = attribution;
 
 export {Control, control};
